docs/EXTENDED-ARCHIVES.md

```diff
@@ -0,0 +1,344 @@
+
+Vendor
+------
+
+> A vendor represents a manufacturer of a device type and potential publisher
+  of official images.
+
+_Some examples of vendors are: Beagleboard, Raspberry Pi
+Foundation, Intel, and Tizen._
+
+Here's an example of a real-world vendor manifest for Beagleboard:
+
+```json
+{
+  "name": "Beagleboard",
+  "url": "http://beagleboard.org",
+  "supportUrl": "http://beagleboard.org/support/",
+  "logoPath": "vendor.svg",
+  "colorScheme": {
+    "backgroundColor": "#535760",
+    "foregroundColor": "#FFFFFF",
+    "primaryColor": "#5793db"
+  }
+}
+```
+
+A vendor may be described with any of the following properties:
+
+### `name (String)`
+
+The display human-friendly name of the vendor.
+
+### `url (String)`
+
+The main url of the vendor, usually the landing page.
+
+### `supportUrl (String)`
+
+The vendor url where users can get general support.
+
+This could be a link to a forum, IRC room, troubleshooting page, support form,
+etc.
+
+### `logoPath (String)`
+
+The path to the vendor's logo in SVG format.
+
+The SVG format requirement ensures that the logo looks good in all types of
+screen resolutions.
+
+### `colorScheme (Object)`
+
+The vendor specific color scheme, if any.
+
+Declaring a vendor-specific color scheme allows a certain degree of branding to
+be applied to the application on the fly as soon as a vendored image is
+selected.
+
+You may declare the following colors, in hexadecimal format:
+
+- `backgroundColor`: The application background color
+- `foregroundColor`: The application foreground color
+- `primaryColor`: The application primary color
+
+Device
+------
+
+> A device represents a specific device type manufactured by a certain vendor.
+
+_Some examples of devices are: Beaglebone Black, Beaglebone Green, Intel Edison,
+and Raspberry Pi 3._
+
+Here's an example of a real-world device manifest for Raspberry Pi 3:
+
+```json
+{
+  "displayName": "Raspberry Pi 3",
+  "arch": "armv7hf",
+  "slug": "raspberrypi3",
+  "slugAliases": [
+    "raspberry-pi3",
+    "rpi3"
+  ],
+  "url": "https://www.raspberrypi.org/products/raspberry-pi-3-model-b/",
+  "photoPath": "raspberry-pi-3-photo.jpg"
+}
+```
+
+A device may be described with any of the following properties:
+
+### `displayName (String)`
+
+The human-friendly name of the device.
+
+### `arch (String[])`
+
+The device architecture.
+
+The current possible values are: `aarch64`, `amd64`, `armv5e`, `armv7hf`,
+`i386`, and `rpi`.
+
+### `slug (String)`
+
+The device slug.
+
+### `slugAliases (String[])`
+
+The device slug aliases, if any.
+
+### `url (String)`
+
+The url to the landing page of the device.
+
+### `photoPath (String)`:
+
+The path to a photo of the device in JPEG format.
+
+### `compatibleWith (String[])`
+
+An array of the device slugs this device is compatible with.
+
+Declaring compatibility allows images that target a specific device to be also
+used in other device types that are known to work with it.
```

Comment:
We certainly have standardised slugs in resin.io, and it would be nice if we can kickstart a kind of device type ontology, but we have to be careful about adding some pragmatic guardrails to that sort of thing because otherwise we end up with the Semantic Web, writing in RDF and OWL-DL.